packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -207,8 +207,8 @@ const DefaultSummaryConfiguration: ISummaryConfiguration = {
     // Snapshot if 1000 ops received since last snapshot.
     maxOps: 1000,
 
-    // Wait 10 minutes for summary ack
-    maxAckWaitTime: 600000,
+    // Wait 2 minutes for summary ack
+    maxAckWaitTime: 120000,
```

Comment:
This will not be enough. This is default value that we use if server did not supply policy.
I believe SPO supplies policy, and while it's sort of violation of contract, I believe client should put an upper bound on it at 2m (to be adjusted based on future feedback). I.e. place where we use it, I'd cap it at 2min in addition (or as a replacement) to this change.

Reply:
I've added in the change to use a maximum limit of 2 minutes. However, this means that the value returned by the server config will not be used in the scenarios where it is greater. Is that okay?

Reply:
Correct. We should continue discussion on long term plan here. I'd say that service should limit their number, but also client should be able to overwrite it. But we will arrive to some long term solution as part of that discussion